woocommerce/woocommerce-ios
# [Custom Amounts M2] Re-structure payment section of order creation

Closes: #10884 


## Description

With this PR we restructure the Payment section of order creation to match the new design ichqhxDjbmZZmvJaJkGhkj-fi-118%3A6643

As this goes directly into production, we don't need feature flag.

## Testing instructions


1. Go to Orders
2. Tap on + to create a new order.
3. Check that the new design matches:

- Empty State (no items added)
- Add products, custom amounts, shipping, coupons, discounts, gift cards to check that it works fine.
- Add customer details and notes to check that it works according to design as well.
- Check that when the order is not editable (e.g. completed) 

## Screenshots


https://github.com/woocommerce/woocommerce-ios/assets/1864060/a8080cb5-7ae0-4b60-9dff-e44c6a829dd6

---
